openreview/venue_request/process/matchingProcess.py

```diff
@@ -36,21 +36,22 @@ def process(client, note, invitation):
     print(traceback.format_exc())
 
     comment_note = openreview.Note(
-        invitation = note.invitation.replace('Paper_Matching_Setup', 'Comment'),
+        invitation = note.invitation.replace('Paper_Matching_Setup', 'Paper_Matching_Setup_Status'),
         forum = note.forum,
         replyto = note.id,
         readers = [conference.get_program_chairs_id()] + [SUPPORT_GROUP],
         writers = [],
         signatures = [SUPPORT_GROUP],
         content = {
-            'title': 'Matching Status [{note_id}]'.format(note_id=note.id),
+            'title': 'Paper Matching Setup Status',
```

Comment:
I think we can remove the `title` field from the invitation. The UI automatically shows the invitation id as status.

Reply:
We need `title` when sending email to the PCs.

Reply:
Could you have a different process function per invitation? I do think we need to customize the email message for each status case.

Reply:
okay